examples/deprecated_flows.ts

```diff
@@ -0,0 +1,165 @@
+import Justifi from "../lib";
+import { randomUUID } from "crypto";
+import { SellerAccount } from "../lib/internal/account";
+import { PaymentCaptureStrategy } from "../lib/internal/payment";
+import { RefundReason } from "../lib/internal/refund";
+
+if (!process.env.CLIENT_ID || !process.env.CLIENT_SECRET) {
+  throw new Error("CLIENT_ID or CLIENT_SECRET missing");
+}
+
+const client = Justifi.client().withCredentials({
+  clientId: process.env.CLIENT_ID,
+  clientSecret: process.env.CLIENT_SECRET,
+});
+
+const runSellerAccountTest = async (): Promise<SellerAccount> => {
```

Comment:
Since this won't be used by the user in the sdk we can just remove it and have the one with the sub account